templates/Layout/DNRoot_alerts.ss

```diff
@@ -79,11 +76,9 @@ alerts:
 	</ul>
 
 	<p>
-		Please contact the <a href="http://helpdesk.silverstripe.com" target="_blank">SilverStripe Operations Team</a>
-		if you would like to add another contact to the list above.
+		For the moment, please raise a <a href="http://helpdesk.silverstripe.com" target="_blank">helpdesk</a> ticket with our operations team and tell us the email addresses that should receive notifications.
```

Comment:
Probably would say something like: "Please raise a <a href="http://helpdesk.silverstripe.com" target="_blank">helpdesk</a> ticket and advise us which email addresses should receive notifications."